### PR TITLE
help me sign up flow bs4 conversion started

### PR DESCRIPTION
**Pull request by kristinmerbach**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit)
- [ ] Tests for the changes have been added (for bugfixes/features), they use let helpers and before blocks
- [ ] For all UI changes, there is cucumber coverage
- [ ] Any endpoint touched in the PR has an appropriate Pundit policy. For open endpoints, reasoning is documented in PR and code
- [ ] Any endpoint modified in the PR only responds to the expected MIME types.
- [ ] For all scripts or rake tasks, how to run it is documented on both the PR and in the code
- [ ] There are no inline styles added
- [ ] There are no inline javascript added
- [ ] There is no hard coded text added/updated in helpers/views/Javascript. New/updated translation strings do not include markup/styles, unless there is supporting documentation
- [ ] Code does not use .html_safe
- [ ] All images added/updated have alt text
- [ ] Doesn’t bypass rubocop rules in any way

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update to a version)

# What is the ticket # detailing the issue?

Ticket: https://www.pivotaltracker.com/story/show/187347115

# A brief description of the changes

Current behavior:

New behavior:

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name:

- [ ] DC
- [ ] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

# AppScan CodeSweep Failure
In the event of a failed check on the AppScan CodeSweep step of our GitHub Actions workflow, please review the False Positive protocol outlined here: appscan_codesweep/CODESWEEP_FALSE_POSITIVES_README.MD

Add all required notes to this section if the failure is a suspected false positive.

### CODE REVIEW

File: app/views/ui-components/v1/modals/_help_with_plan.html.erb [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+          <% end %>
+          <div class="hide" id="help_requestor"><%= @person.try(:id) %></div>
+          <div class="hide" id="help_requestor_email"><%= current_user.oim_id %></div>
+          <div class="hide" id="help_type"></div>
+        </div>
+        <div id="inbox" class="help_reset">
+          <div class="col-md-10 col-md-12 col-lg-12"></div>
+        </div>
+        <div class="col-md-3">
+          <div class="col-md-8 btn btn-default hide" id="back_to_help">Back</div>
```
Comment (ATBull81) [inline app/views/ui-components/v1/modals/_help_with_plan.html.erb:93]: We should l10n "Back", think it already exists in the translations files.

File: components/benefit_sponsors/app/views/benefit_sponsors/profiles/broker_agencies/broker_agency_profiles/_broker_help.html.erb [hunk 2]
```diff
@@ -2,6 +2,52 @@
...
+        <% else %>
+          <%= l10n("broker_agencies.profiles.no_address")%>
+        <% end %>
+      </dd>
+    </dl>
+    <dl class="parent my-2">
+      <dt><%= l10n("broker_agencies.profiles.new_clients")%>:</dt>
+      <dd><%= broker_agency_profile.accept_new_clients? ? 'Yes' : 'No' %></dd>
+      <dt><%= l10n("broker_agencies.profiles.weekend_evening_hours")%>:</dt>
+      <dd><%= broker_agency_profile.working_hours? ? 'Yes' : 'No' %></dd>
```
Comment (ATBull81) [inline components/benefit_sponsors/app/views/benefit_sponsors/profiles/broker_agencies/broker_agency_profiles/_broker_help.html.erb:43]: same l10n request for yes/no

File: components/benefit_sponsors/app/views/benefit_sponsors/profiles/broker_agencies/broker_agency_profiles/_staff_table.html.erb [hunk 3]
```diff
@@ -1,20 +1,43 @@
...
+    </tr>
+  </thead>
+  <tbody>
+    <% @staff.each do |staff| %>
+      <% organization = BenefitSponsors::Organizations::Organization.where(:"profiles._id" => staff.broker_role.benefit_sponsors_broker_agency_profile_id).first %>
+      <tr>
+        <td class='col-sm-4'><%= staff.first_name + ' ' + staff.last_name %></td>
+        <td class='col-sm-3'><%= organization.legal_name %></td>
+        <td class='col-sm-3'><% market_kind = organization.broker_agency_profile.market_kind.to_s %>
+          <%= (market_kind.eql? "both") ? "SHOP & Individual" : market_kind.titleize %><%= l10n("market")%></td>
```
Comment (ATBull81) [inline components/benefit_sponsors/app/views/benefit_sponsors/profiles/broker_agencies/broker_agency_profiles/_staff_table.html.erb:17]: l10n "SHOP & Individual" here and on line 38

**Review by ATBull81 — APPROVED** [review]
~~Looks good, some small nits on l10n but I'm alright with sweeping back through this sign up flow (and other areas) and improving in a later PR.~~

Should've read ticket expectations first. Yeah just add those translations and I think its good!

File: app/views/ui-components/v1/modals/_help_with_plan.html.erb (hunk 1 above)
Comment (kristinmerbach) [reply app/views/ui-components/v1/modals/_help_with_plan.html.erb]: resolved

File: components/benefit_sponsors/app/views/benefit_sponsors/profiles/broker_agencies/broker_agency_profiles/_broker_help.html.erb (hunk 2 above)
Comment (kristinmerbach) [reply components/benefit_sponsors/app/views/benefit_sponsors/profiles/broker_agencies/broker_agency_profiles/_broker_help.html.erb]: done! should be ready for reapproval

**Review by charlienparker — APPROVED** [review]

**Review by ATBull81 — APPROVED** [review]
LGTM after the conflict is taken care of

**Pull request merged**